src/linker/Linker.Steps/ResolveFromXmlStep.cs

```diff
@@ -344,19 +343,20 @@ protected virtual void ProcessField (TypeDefinition type, XPathNodeIterator iter
 		void ProcessFieldSignature (TypeDefinition type, string signature)
 		{
 			FieldDefinition field = GetField (type, signature);
-			MarkField (type, field, signature);
+			if (field == null) {
+				AddUnresolveMarker (string.Format ("T: {0}; F: {1}", type, signature));
```

Comment:
Just related question: I can't find any usage of this. We store the information about unresolved things, but we don't write it anywhere. Should we at least output these as Normal/Diag messages?
Also why do we store these in `ResolveStep` - is this an extensibility point, or just left-over?

Reply:
I think this is some legacy code, made #1123 to track the removal